### PR TITLE
Adding docker compose

### PR DESCRIPTION
**Pull request by TeeronHT**
This adds the docker-compose.yml file as well as the DockerFile

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export const INIT_AMOUNT = Number(process.env.INIT_AMOUNT) || 0.01
 
 // Portal Setup
 export const PORTAL_API_URL =
-  process.env.PORTAL_API_URL || 'https://api.portalhq.io'
+  process.env.PORTAL_API_URL || 'https://api-staging.portalhq.io'
```
Comment (dscrobonia) [inline src/config.ts:13]: any reason for updating this to be staging?

**Review by dscrobonia — CHANGES_REQUESTED** [review]
Can you add an update to the README explaining how to use the docker-compose command to spin everything up?

File: src/config.ts (hunk 1 above)
Comment (TeeronHT) [reply src/config.ts:13]: I had just done that locally and accidentally pushed it, apologies.

Comment (dscrobonia) [reply src/config.ts]: all good!

**Review by dscrobonia — APPROVED** [review]
LGTM!

**Pull request merged**